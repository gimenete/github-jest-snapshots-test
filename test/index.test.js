it('renders correctly', () => {
  const pack1 = require('../package.json')
  expect(pack1).toMatchSnapshot();
});
