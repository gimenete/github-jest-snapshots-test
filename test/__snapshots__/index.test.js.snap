// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "author": "",
  "bugs": Object {
    "url": "https://github.com/gimenete/github-jest-snapshots-test/issues",
  },
  "description": "Another description",
  "devDependencies": Object {
    "jest": "^24.0.0",
  },
  "homepage": "https://github.com/gimenete/github-jest-snapshots-test#readme",
  "keywords": Array [],
  "license": "ISC",
  "main": "index.js",
  "name": "github-jest-snapshots-test",
  "repository": Object {
    "type": "git",
    "url": "git+https://github.com/gimenete/github-jest-snapshots-test.git",
  },
  "scripts": Object {
    "test": "jest",
  },
  "version": "1.0.0",
}
`;
